test/common/upstream/cluster_manager_impl_test.cc

```diff
@@ -359,7 +359,7 @@ TEST_F(ClusterManagerImplTest, OriginalDstLbRestriction) {
 
 TEST_F(ClusterManagerImplTest, OriginalDstLbRestriction2) {
   const std::string yaml = R"EOF(
-static_resources:
+ static_resources:
```

Comment:
How come we have these gratuitous whitespace changes in this file?

Reply:
@htuch for me is result of invoking `tools/check_format.py fix` in local dev environment. Could manually revert format changes later today.